src/PKSim.CLI/Program.cs

```diff
@@ -5,74 +5,82 @@
 using OSPSuite.Utility.Container;
 using PKSim.CLI.Commands;
 using PKSim.CLI.Core.Services;
-using PKSim.CLI.Services;
-using ILogger = OSPSuite.Core.Services.ILogger;
+//using PKSim.CLI.Services;
+using Serilog.Extensions.Logging;
+using PKSim.Infrastructure.Services;
+using IOSPLogger = OSPSuite.Core.Services.ILogger;
 
 namespace PKSim.CLI
 {
-   [Flags]
-   enum ExitCodes
-   {
-      Success = 0,
-      Error = 1 << 0,
-   }
+  [Flags]
```

Comment:
@abdelr Looks like you are using 2 indent instead of 3
https://github.com/Open-Systems-Pharmacology/Suite/blob/develop/CODING_STANDARDS.md

Reply:
@msevestre Corrected already